martian-lab/tests/split_test/expected/SUM_SQUARES/SUM_SQUARES/fork0/split/_stage_defs

```diff
@@ -2,25 +2,29 @@
   "chunks": [
     {
       "__mem_gb": 1,
+      "__special": null,
```

Comment:
This should not be written if `special` was unset.  Please fix.

Reply:
https://github.com/martian-lang/martian-rust/pull/518